<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertdata</class>
 <widget class="QMainWindow" name="insertdata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{background-color: rgb(255, 249, 243);
border-radius:20px;border:4px solid #b38144;
}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;幼圆&quot;;
color: rgb(156, 118, 57);</string>
    </property>
    <property name="text">
     <string>试题录入</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:16px;border:3px solid #b38144;
font: 14pt &quot;幼圆&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="puta">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>199</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:16px;border:3px solid #b38144;
font: 14pt &quot;幼圆&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="putb">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>269</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:16px;border:3px solid #b38144;
font: 14pt &quot;幼圆&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="putc">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>339</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:16px;border:3px solid #b38144;
font: 14pt &quot;幼圆&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="putd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>409</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:16px;border:3px solid #b38144;
font: 14pt &quot;幼圆&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitdata">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>532</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:16px;border:3px solid #b38144;
font: 20pt &quot;X-妞妞手迹&quot;;</string>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>51</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;幼圆&quot;;</string>
       </property>
       <property name="text">
        <string>题目</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>540</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
    </property>
    <property name="text">
     <string>答案为：</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>540</y>
      <width>144</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioA">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioB">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioC">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioD">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="closewindow">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/bgrand/img/insertclose.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minwindow">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/bgrand/img/minsize.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
